supervisor/options.py

```diff
@@ -393,6 +393,38 @@ def import_spec(self, spec):
             # this causes a DeprecationWarning on setuptools >= 11.3
             return ep.load(False)
 
+    def read_include_config(self, fp, parser, expansions):
+        if parser.has_section('include'):
+            parser.expand_here(self.here)
+            if not parser.has_option('include', 'files'):
+                raise ValueError(".ini file has [include] section, but no "
+                "files setting")
+            files = parser.get('include', 'files')
+            files = expand(files, expansions, 'include.files')
+            files = files.split()
+            if hasattr(fp, 'name'):
+                base = os.path.dirname(os.path.abspath(fp.name))
+            else:
+                base = '.'
+            for pattern in files:
+                pattern = os.path.join(base, pattern)
+                filenames = glob.glob(pattern)
+                if not filenames:
+                    self.parse_warnings.append(
```

Comment:
`self.parse_warnings` and `self.parse_infos` are undefined in the context of `ClientOptions`.

Reply:
Marking this as resolved, the commit above fixes it, I believe